src/topology/category/Profinite.lean

```diff
@@ -133,4 +133,15 @@ lemma CompHaus.to_Profinite_obj' (X : CompHaus) :
 instance Profinite.to_CompHaus.reflective : reflective Profinite.to_CompHaus :=
 { to_is_right_adjoint := ⟨CompHaus.to_Profinite, Profinite.to_Profinite_adj_to_CompHaus⟩ }
 
+noncomputable
+instance Profinite.to_Top.reflective : reflective (Profinite_to_Top : Profinite ⥤ Top) :=
+reflective.comp Profinite.to_CompHaus CompHaus_to_Top
+
+noncomputable
+instance Profinite.to_Top.creates_limits : creates_limits Profinite_to_Top :=
```

Comment:
I think it would be nicer to have `creates_limits Profinite.to_CompHaus` (and `creates_limits CompHaus_to_Top` if it's not already there), and then prove this one by composition.

Reply:
I added those two, but don't you think we should have the reflective instance from Profinite to Top? 
Would there be any benefit to proving creates_limits by composition vs. by reflectivity (which is obtained by composition)?

Reply:
I agree we should have the reflective instance from Profinite to Top! I think there's no difference in the way creates_limits is shown (except maybe one version might be an automatic instance so it doesn't need to be explicitly written - I might be misremembering)

Reply:
I don't think any of these instances are automatic in this case.